fili-generic-example/src/main/java/com/yahoo/wiki/webservice/data/config/metric/FiliMetricName.java

```diff
@@ -0,0 +1,41 @@
+package com.yahoo.wiki.webservice.data.config.metric;
+
+import com.yahoo.bard.webservice.data.config.names.ApiMetricName;
+import com.yahoo.bard.webservice.data.time.TimeGrain;
+import com.yahoo.bard.webservice.util.EnumUtils;
+
+import java.util.List;
+import java.util.Locale;
+
+/**
+ * Created by kevin on 3/7/2017.
+ */
+public class FiliMetricName implements ApiMetricName {
+    private String apiName;
```

Comment:
This may be better named FiliApiMetricName because it extends ApiMetricName.
(I'm experiencing deja vu, I thought I already made this comment elsewhere)